docs/UserGuide.md

```diff
@@ -52,9 +52,9 @@ If you can type fast, TA<sup>2</sup> can get your contact and assignment managem
  * Result Message Box: Displays a success message if your command has been executed, or an error message if your command 
```

Comment:
Displays the result of your commands. <- Perhaps this is better?

Command result box maybe can change to "This is where you can type your commands."